app/Exceptions/Handler.php

```diff
@@ -61,14 +61,14 @@ public function register(): void
             }
         });
 
-        $this->renderable(function (Exception $exception) {
-            if ($exception instanceof HttpException && $exception->getStatusCode() >= 500) {
-                return $this->handle500Error($exception);
-            }
 
+        $this->renderable(function (Exception $exception) {
             if ($exception instanceof HttpException && $exception->getStatusCode() === 404) {
                 return $this->handle404Error($exception);
             }
+            if (!$exception instanceof HttpException) {
+                return $this->handle500Error($exception);
+            }
```

Comment:
```suggestion
        $this->renderable(function (Exception $exception) {
            if (!$exception instanceof HttpException) {
                return $this->handle500Error($exception);
            }
            if ($exception->getStatusCode() === 404) {
                return $this->handle404Error($exception);
            }
            if ($exception->getStatusCode() >= 500) {
                return $this->handle500Error($exception);
            }            
```

We still have to check http exceptions that are 500.